lib/applications/src/launcher.cpp

```diff
@@ -19,20 +19,25 @@
  */
 std::string getFormatedDate()
 {
-    uint16_t day_ = GSM::days;
-    uint16_t day = GSM::days;
-    uint16_t month = GSM::months;
-    uint16_t year = GSM::years;
+    int16_t day_ = Gsm::Time::getDay();
+    int16_t day = Gsm::Time::getDay();
+    int16_t month = Gsm::Time::getMonth();
+    int16_t year = Gsm::Time::getYear();
+
+    if(day_ == -1) day_ = 1;
+    if(day == -1) day = 1;
+    if(month == -1) month = 1;
+    if(year == -1) year = 1970;
 
     std::string dayName = daysOfWeek[(day+=month<3?year--:year-2,23*month/9+day+4+year/4-year/100+year/400)%7];
     std::string monthName = daysOfMonth[month==0?1:(month-1)];
 
-    return dayName + " " + std::to_string(GSM::days) + " " + monthName;
+    return dayName + " " + std::to_string(Gsm::Time::getDay()) + " " + monthName;
```

Comment:
On peut directement utiliser la variable `day`
```suggestion
    return dayName + " " + std::to_string(day) + " " + monthName;
```